cmd/thanos/query.go

```diff
@@ -85,7 +85,9 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 	dnsSDInterval := modelDuration(cmd.Flag("store.sd-dns-interval", "Interval between DNS resolutions.").
 		Default("30s"))
 
-	enableAutodownsampling := cmd.Flag("query.auto-downsampling", "Enable automatic adjustment (step / 5) to what source of data should be used in store gateways if no max_source_resolution param is specified. ").
+	unhealthyStoreTimeout := modelDuration(cmd.Flag("store.unhealthy-timeout", "Timeout in minutes before a store is cleaned from the store UI page.").Default("5m"))
```

Comment:
nit: there is no mention that the cleaned up store needs to be unhealthy in the help just in the name of the flag. Maybe could e good to clear that out in there also?